cli/cmd/root.go

```diff
@@ -149,4 +117,9 @@ func initViper() {
 	viper.AutomaticEnv()
 	viper.SetEnvPrefix("CQ")
 	viper.SetEnvKeyReplacer(strings.NewReplacer("-", "_"))
+	viper.SetConfigName("config")    // name of config file (without extension)
```

Comment:
Where do we use these settings?

Reply:
It will give people the ability to turn of analytics permanently without passing it on each run

Reply:
Can you share an example on how this config file will look like? Not sure we'd want to support another configuration API, and if so, maybe call it `cq_config.yaml`? Or `cloudquery.yaml`.

Reply:
This is just standard viper (https://github.com/spf13/viper#reading-config-from-ioreader) so users will be able to use 

```
telemetry-level: "off"
```

for flags that are passed to viper (like telemetry)

Reply:
I also went further ahead and I removed viper library because it's not a great one, also we just use environment variable for some flags and if we use viper we loose the built-in cobra flag verification and need to reimplement. Moreover this also causes a confusion on when we should use viper and/or the variable bound via the flags because viper is only one way binding.

Now should be pretty simple and I just shaved up 1MB from our CLI because viper imports hcl,toml,yaml2,yaml3,ini. I think the big one was hcl as it's basically another language.